src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Appeal.cshtml

```diff
@@ -57,32 +57,117 @@
 
                             <partial name="_GatewayOutcome" for="@Model.GatewayOutcome" />
 
+                            @if (!Model.HasFinalOutcome && !Model.IsGatewayActionPendingConfirmation)
+                            {
+                                <span das-validation-for="ApproveGateway" class="govuk-error-message"></span>
+
+                                <div class="govuk-radios govuk-radios--inline govuk-!-margin-bottom-6">
+                                    <div class="govuk-radios__item">
+                                        <input class="govuk-radios__input"
+                                               type="radio"
+                                               value="@ApprovalStatus.Approve"
+                                               asp-for="ApproveGateway" />
+                                        <label class="govuk-label govuk-radios__label" for="ApproveGateway">
+                                            Approve this outcome
+                                        </label>
+                                    </div>
+                                    <div class="govuk-radios__item">
+                                        <input class="govuk-radios__input"
+                                               type="radio"
+                                               id="OptionApproveGatewayNo"
+                                               value="@ApprovalStatus.Overturn"
+                                               asp-for="ApproveGateway"
+                                               data-module="das-show-hide-controls"
+                                               data-aria-controls="conditional-UnsuccessfulExternalText" />
+                                        <label class="govuk-label govuk-radios__label" for="OptionApproveGatewayNo">
+                                            Overturn this outcome
+                                        </label>
+                                    </div>
+                                </div>
+                            }
                         </fieldset>
                     </div>
```

Comment:
This is being highlighted as a change. Aren't we supposed to be on a readonly view?